test/addons/rook-pool/storage-class.yaml

```diff
@@ -6,6 +6,8 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
     name: rook-ceph-block
+    labels:
+        ramendr.openshift.io/storageid: rook-ceph-storage-id
```

Comment:
Ideally the storage ID should be different in the 2 clusters, as this is not a MetroDR setup. So in this case, we should "maybe" add the value with the cluster name to start with.

If we specify the same storage ID, as we develop peer classes at the hub DRPolicy would assume that this is a metro setup in the future.

Reply:
This will need attention, if not now then pretty soon as we add code for update DRPolicy.status.peerLists as per https://github.com/RamenDR/ramen/pull/1403

Does not block this PR as such, but something to correct down the line.